.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -1,28 +1,14 @@
 name: Docker
 
 on:
-  push:
```

Comment:
Is this intended?

Reply:
It's taken from here (https://github.com/Lightning-AI/lightning/pull/13716/commits/11c58e01d4813e740bae04ca2f4f3cee47c3cc8a), I fully intend to revert that commit

Reply:
Reverted now